libs/api/domains/finance/src/lib/api-domains-finance.resolver.ts

```diff
@@ -29,11 +29,12 @@ import {
   PaymentScheduleDetailModel,
   PaymentScheduleModel,
 } from './models/paymentSchedule.model'
+import { DebtLessCertificateModel } from './models/debtLessCertificate.model'
 import { DebtStatusModel } from './models/debtStatus.model'
 import { GetFinancePaymentScheduleInput } from './dto/getFinancePaymentSchedule.input'
 
 @UseGuards(IdsUserGuard, ScopesGuard)
-@Scopes(ApiScope.financeOverview)
+@Scopes(ApiScope.financeOverview, ApiScope.internal)
```

Comment:
Should this scope rather be added to the getDebtLessCertificate() instead of the whole resolver? These scopes are there for a reason and I'm afraid this will give everyone with internal scope access to all the resolver functions. Maybe @saevarma can clarify this?